### PR TITLE
Update acceptStudent function

### LINKED ISSUE #3: Create function to accept a student name acceptStudent, that accepts (ID:string, FullName:string, Age:int, Address:string)
- The status of the student is set to "Applying."

### PR DESCRIPTION
**Pull request by amsolaiman**
- Data is store as JSON object as requested.

### CODE REVIEW

**Comment by amsolaiman** [comment]
- The acceptStudent function is updated to store values as JSON object.

File: main.js [hunk 1]
```diff
@@ -1,15 +1,59 @@
...
+
+async function deleteStudent(id){
+    await db.del(id)
+    db.get(id, function(err, value){
+        if(err){
+            console.log('Data is deleted');
+        } else{
+            console.log(value); 
+        }
+    })
```
Comment (fencim) [inline main.js:58]: These changes are not part of the issue... reserve this change for the other developers other

**Review by fencim — CHANGES_REQUESTED** [review]
remove un-related changes

File: main.js (hunk 1 above)
Comment (amsolaiman) [reply main.js:58]: Noted sir. We'll try to start from scratch again.

**Pull request closed without merging**